tools/update_gradle_wrapper.sh

```diff
@@ -10,12 +10,22 @@ HERE=`pwd`
 
 cd "$(dirname $0)/.."
 
+pushd .
+
 for i in $(find $(pwd) -type f -name gradlew); do
     cd $(dirname $i)
     pwd
     ./gradlew wrapper
     sed -E -i '' s/-bin\\.zip\$/-all.zip/ gradle/wrapper/gradle-wrapper.properties
 done
 
+popd
+
+sed -i '' '/^APP_ARGS=/a\
+# Realm'"'"'s work-around for a bug in Gradle 4.1\
```

Comment:
No. we can't use ' in single quote string. `'"'"'` closes single quote string then open double quote string that contains one single quote in it the reopen single quote string again.
These three strings are concatenated.

Is there any efficient way to contain single quote in single quote string?